From 6f32ce275e0ba3e80b5f9661743dc5c682591372 Mon Sep 17 00:00:00 2001
From: Daniel Stone <daniels@collabora.com>
Date: Tue, 14 Dec 2021 14:52:45 +0000
Subject: [PATCH 01/60] tests/xwayland: Remove manual timeout implementation

One of the best things about a real test framework is that they handle
timeouts for you.

Signed-off-by: Daniel Stone <daniels@collabora.com>
Reported-by: Simon Ser <contact@emersion.fr>
---
 tests/xwayland-test.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/xwayland-test.c b/tests/xwayland-test.c
index 665f6ea7..7017eb00 100644
--- a/tests/xwayland-test.c
+++ b/tests/xwayland-test.c
@@ -108,7 +108,6 @@ TEST(xwayland_client_test)
 	XSelectInput(display, window, ExposureMask);
 	XMapWindow(display, window);
 
-	alarm(4);
 	while (1) {
 		XNextEvent(display, &event);
 		if (event.type == Expose)
-- 
2.20.1

